udata_hydra/crawl.py

```diff
@@ -307,10 +314,40 @@ async def crawl_urls(to_parse):
 
 def get_excluded_clause():
     return " AND ".join(
-        [f"catalog.url NOT LIKE '{p}'" for p in config.EXCLUDED_PATTERNS]
+        [f"catalog.url NOT LIKE '{p}'" for p in config.EXCLUDED_PATTERNS] +
+        [
+            "catalog.deleted = False",
+            "(catalog.status <> 'crawling' OR catalog.status IS NULL)"
```

Comment:
why `<>` and not `!=` ?

Reply:
It had to do with me struggling with NULL comparison, replaced with `!=` :)